### PR TITLE
Allows `install_binder.sh` to be independently executable

### PR DESCRIPTION
**Pull request by eitsupi**
As mentioned in the following comment, currently `install_binder.sh` cannot be executed unless `install_python.sh` has been executed in advance.
This is different from the behavior of other installation scripts and should be changed so that it can be executed independently.
https://github.com/rocker-org/rocker-versioned2/pull/429#discussion_r855956332

### CODE REVIEW

File: scripts/install_binder.sh [hunk 1]
```diff
@@ -1,31 +1,54 @@
...
+# https://github.com/rocker-org/rocker-versioned2/issues/428
 PATH=/opt/pyenv/bin:${PATH}
-
-# And set ENV for R! It doesn't read from the environment...
-echo "PATH=${PATH}" >>"${R_HOME}/etc/Renviron.site"
 echo "export PATH=${PATH}" >>"${WORKDIR}/.profile"
 
 cd "${WORKDIR}"
 ## This gets run as user
 sudo -u "${NB_USER}" python3 -m venv "${PYTHON_VENV_PATH}"
```
Comment (eitsupi) [inline scripts/install_binder.sh:43]: I do not understand the need for this line.
My understanding is that this environment is not used within this script...

Comment (eitsupi) [reply scripts/install_binder.sh:45]: It seems that `install_python.sh` also has unused code related to `venv` or `virtualenv`.
Do you think it is OK to delete these all together as well?

Comment (cboettig) [reply scripts/install_binder.sh:45]: probably. do you want to tackle updates to install_python.sh here?

Comment (eitsupi) [reply scripts/install_binder.sh:45]: This PR set up tests for each script and it make it easier future changes, so I would like to remove the virtual envs at another time.
I would like to merge this PR into #441 first to make sure it passes all tests and then into the default branch.

**Pull request merged**